lib/mariaex/row_parser.ex

```diff
@@ -271,6 +276,10 @@ defmodule Mariaex.RowParser do
     decode_bin_rows(rest, fields, null_bitfield, [{{year, month, day}, {hour, min, sec, msec}} | acc])
   end
 
+  defp decode_geometry(<<25::8-little, srid::32-little, _endian::8-little, 1::32-little, x::little-float-64, y::little-float-64, rest::bits >>, fields, null_bitfield, acc) do
```

Comment:
If a service speaks MySQL it may not be MySQL so lets ensure we add little endian :).